src/services/AppsApiService.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+import config from '../config';
+
+export default class AppsApiService {
+  static async get(authToken) {
+    const url = AppsApiService.#buildUrl();
+
+    return axios.get(url, {
+      headers: {
+        Authentication: authToken,
+      },
+    });
+  }
+
+  static async getById(authToken, { id }) {
+    const url = AppsApiService.#buildUrl();
+
+    return axios.get(url.href + `/${id}`, {
+      headers: {
+        Authentication: authToken,
+      },
+    });
+  }
+
+  static async updateById(authToken, { id, name }) {
+    const url = AppsApiService.#buildUrl();
+
+    return axios.put(url.href + `/${id}`, {
+      headers: {
+        Authentication: authToken,
+      },
+      data: {
+        name,
+      },
+    });
+  }
+
+  static async deleteById(authToken, { id }) {
+    const url = AppsApiService.#buildUrl();
+
+    return axios.delete(url.href + `/${id}`, {
+      headers: {
+        Authentication: authToken,
+      },
+    });
+  }
+
+  static async post(authToken, { name }) {
+    const url = AppsApiService.#buildUrl();
+
+    return axios.post(url, {
+      headers: {
+        Authentication: authToken,
+      },
+      data: {
+        name,
+      },
```

Comment:
This code not works correct, it's not send "name" field.
You can do like this and it should work fine
```suggestion
    return axios.post(
      url,
      { name },
      {
        headers: {
          Authentication: authToken,
        },
      }
```